### ISSUE TITLE
CUDA out of memory

### ISSUE THREAD

**Issue by ynnirc** [sensity-ai/dot#53]
### Description:

When running dot with SimSwap using a GPU, I am getting this error:
```
RuntimeError: CUDA out of memory. Tried to allocate 7.15 GiB (GPU 0; 8.00 GiB total capacity; 4.42 GiB already allocated; 544.09 MiB free; 4.50 GiB reserved in total by PyTorch)
```

No app is using the GPU besides dot. The same setup used to work fine, but once after I rebooted my PC, this problem started showing up. I don't remember making any changes that could impact it. Here is what I've tried without any success:
- Reinstall pytorch and related packages
- Reinstall CUDA
- Reboot

### Additional info:
- torch==1.9.0+cu111, torchvision==0.10.0+cu111
- CUDA 1.11
- Ryzen 5 3600x
- RTX 3070 (8GB RAM)

**Comment by Ghassen-Chaabouni** [comment]
To run dot, you only need 4GB of GPU memory. Please use `nvidia-smi` before running dot and double check if there is any other app using GPU.

**Comment by ynnirc** [comment]
> To run dot, you only need 4GB of GPU memory. Please use `nvidia-smi` before running dot and double check if there is any other app using GPU.

Here is the output:
```
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 527.41       Driver Version: 527.41       CUDA Version: 12.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name            TCC/WDDM | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA GeForce ... WDDM  | 00000000:0A:00.0  On |                  N/A |
|  0%   38C    P0    47W / 240W |    584MiB /  8192MiB |      2%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A      1608    C+G   ...Client\ui\.... .......exe    N/A      |
|    0   N/A  N/A      8648    C+G   ...er_engine\............exe    N/A      |
|    0   N/A  N/A      9036    C+G   ...gram Desktop\.........exe    N/A      |
|    0   N/A  N/A      9508    C+G   C:\Windows\.........exe         N/A      |
|    0   N/A  N/A     11224    C+G   ...5n1h2txyewy\..........exe    N/A      |
|    0   N/A  N/A     11672    C+G   ...um\Application\.......exe    N/A      |
|    0   N/A  N/A     13248    C+G   ...2txyewy\..............exe    N/A      |
|    0   N/A  N/A     14928    C+G   ...8bbwe\................exe    N/A      |
+-----------------------------------------------------------------------------+
```

As far as I can tell, there is no GPU usage.

**Comment by Ghassen-Chaabouni** [comment]
@ynnirc are you using the default images in the [data](https://github.com/sensity-ai/dot/tree/main/data) folder? If not can you post the images that you are using.
Did you make any changes to the repo?
Also, what command are you using to run SimSwap?

**Comment by ynnirc** [comment]
Thanks for the response @Ghassen-Chaabouni! I've dropped a default image into the data folder and it worked. Interestingly, the images that previously resulted in this `CUDA out of memory` now work fine when there is a default image in the folder.

I'm afraid I can't provide the image for privacy reasons, but I've found out that after compressing it, dot processes it without any issues. The photo was 1.63 MB before compression and became 108 KB after.
